flink-runtime/src/main/java/org/apache/flink/runtime/util/config/memory/ManagedMemoryUtils.java

```diff
@@ -155,4 +157,20 @@ public static double getFractionRoundedDown(final long dividend, final long divi
                         BigDecimal.ROUND_DOWN)
                 .doubleValue();
     }
+
+    public static void validateUseCaseWeightsNoConflict(
+            Map<ManagedMemoryUseCase, Integer> existingOperatorScopeUseCaseWeights,
+            Map<ManagedMemoryUseCase, Integer> newOperatorScopeUseCaseWeights) {
```

Comment:
nit:
Do we need to identify the two maps (in both arguments and the error message) by "existing" and "new"? From the perspective of this util method, it should make no difference which one is the existing and which one is the new.

Reply:
Good catch! Agree, I will address this during merging the PR.